tripy/tests/spec_verification/test_dtype_constraints.py

```diff
@@ -181,3 +181,63 @@ def test_neg_dtype_constraints(test_data):
         api_call_locals, namespace = _run_dtype_constraints_subtest(test_data)
         if isinstance(api_call_locals[RETURN_VALUE], tp.Tensor):
             assert api_call_locals[RETURN_VALUE].dtype == namespace[return_dtype]
+
+
+def get_all_possible_verif_ops():
```

Comment:
Another approach could be to check all public APIs, and then for classes, check their methods. We can then filter those on whether any of the type annotations include `tp.Tensor`.